po/it.po

```diff
@@ -686,19 +678,16 @@ msgstr ""
 "invece delle 3.18 e 3.20+"
 
 #: plugins/theme_oomox/oomox_plugin.py:107
-#, fuzzy
 msgid "Textbox Caret"
-msgstr "Testo della casella di testo"
+msgstr "Cursore della casella di testo"
 
 #: plugins/theme_oomox/oomox_plugin.py:113
-#, fuzzy
 msgid "BiDi Textbox Caret"
-msgstr "Testo della casella di testo"
+msgstr "Cursore della casella di testo BiDi"
```

Comment:
is BiDi shortening makes sense in italian?

Reply:
I need to know what BiDi is. I did not find in Google.

Reply:
Bi-Drectional